pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -1111,6 +1112,26 @@ func updateACLConfiglet(aclArgs types.AppNetworkACLArgs, oldACLs []types.ACE, AC
 			aclArgs.BridgeName, aclArgs.VifName, aclArgs.AppIP)
 		return oldRules, nil
 	}
+
+	// Before adding new rules, clear flows if any created matching the old rules
+	var family netlink.InetFamily = syscall.AF_INET
+	if aclArgs.IPVer == 4 {
+		family = syscall.AF_INET
+	} else {
+		family = syscall.AF_INET6
+	}
+	srcIP := net.ParseIP(aclArgs.AppIP)
```

Comment:
Is AppIP 0.0.0.0 when not set, or does it make sense to check for an empty string here?

Reply:
I'll change ConntrackDeleteIPSrc to accept empty string or 0.0.0.0 as string to match all flows.